camel/storages/vectordb_storages/milvus.py

```diff
@@ -0,0 +1,360 @@
+
+        Raises:
+            RuntimeError: If there is an error during the deletion process.
+        """
+
+        op_info = self._client.delete(collection_name=self.collection_name,
+                                      pks=ids, **kwargs)
+        logger.debug(f"Successfully deleted vectors in Milvus: {op_info}")
+
+    def status(self) -> VectorDBStatus:
+        r"""Retrieves the current status of the Milvus collection. This method
+        provides information about the collection, including its vector
+        dimensionality and the total number of vectors stored.
+
+        Returns:
+            VectorDBStatus: An object containing information about the
+                collection's status.
+        """
+        status = self._get_collection_info(self.collection_name)
+        return VectorDBStatus(
+            vector_dim=status["vector_dim"],
+            vector_count=status["vector_count"],
+        )
+
+    def query(
+        self,
+        query: VectorDBQuery,
+        **kwargs: Any,
+    ) -> List[VectorDBQueryResult]:
+        r"""Searches for similar vectors in the storage based on the provided
+        query.
+
+        Args:
+            query (VectorDBQuery): The query object containing the search
+                vector and the number of top similar vectors to retrieve.
+            **kwargs (Any): Additional keyword arguments.
+
+        Returns:
+            List[VectorDBQueryResult]: A list of vectors retrieved from the
+                storage based on similarity to the query vector.
+        """
+        search_result = self._client.search(
+            collection_name=self.collection_name,
+            data=[query.query_vector],
+            limit=query.top_k,
+            output_fields=['vector', 'payload'],
+            **kwargs,
+        )
+        query_results = []
+        for point in search_result:
+            query_results.append(
+                VectorDBQueryResult.construct(  # type: ignore
+                    similarity=(1 - point[0]['distance']),
+                    id=str(point[0]['id']),
+                    payload={'message': point[0]['entity'].get('payload')},
+                    vector=point[0]['entity'].get('vector')))
+
+        return query_results
+
+    def clear(self) -> None:
+        r"""Removes all vectors from the Milvus collection. This method
+        deletes the existing collection and then recreates it with the same
+        schema to effectively remove all stored vectors.
+        """
+        self._delete_collection(self.collection_name)
+        self._create_collection(collection_name=self.collection_name)
+
+    @property
+    def client(self) -> Any:
+        r"""Provides direct access to the Milvus client. This property allows
+        for direct interactions with the Milvus client for operations that are
+        not covered by the `MilvusStorage` class.
+
+        Returns:
+            Any: The Milvus client instance.
+        """
```

Comment:
there's no specific type for Milvus client instance, I think `Any: The Milvus client instance.` has provided information of what will be returned

Reply:
I think the expected type is `MilvusClient`, could you please check it?

Ref, from the [official doc](https://milvus.io/docs/using_milvusclient.md):
``` python
from pymilvus import MilvusClient

client = MilvusClient(
    collection_name="qux",
    uri="http://localhost:19530",
    vector_field="float_vector", 
    # pk_field= "id", # If you wanted to provide your own PK
    overwrite=True,
)
```

Reply:
Sorry for the misunderstanding,  here I mean there's no base type for Milvus client. The goal to set type is to provide more information to the users,  with`Any: The Milvus client instance.` we already can do it. If we want to set `def client(self) -> MilvusClient:` then I have to do `from pymilvus import MilvusClient` outside the class, but this can be a heavy solution compared with import it within class. From my side I feel there's no need to do it